js/pickup.js

```diff
@@ -1,5 +1,13 @@
 export const PickupAPI = superclass =>
     class extends superclass {
+        /**
+         * Schedules a pickup for a parcel.
+         *
+         * @param {Object} payload The payload object according to the DHL API standards.
+         * @param {Object} options An object of options to configure the request.
+         * @returns {Object} The HTTP response object.
+         * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section/operations-pickup-exp-api-pickups

```

Comment:
That link takes you to the same place as just `"https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section"`. So let's use this one